NotifyMe/signals.py

```diff
@@ -0,0 +1,29 @@
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+from .models.user import User
+from .models.subscriptionPlan import SubscriptionPlan
+from asgiref.sync import async_to_sync
+from channels.layers import get_channel_layer
+
+
+@receiver(post_save, sender=SubscriptionPlan)
```

Comment:
why are sending notification while new instance of ms table  is added

Reply:
When a new type of plan will be updated in ms_subscription_table then all the users will fet notification that new plan is present now.

Reply:
it is not standard to do any operation when ever we change or update in master table because we always add the values directly in the database